featuretools/tests/primitive_tests/test_max_consecutive.py

```diff
@@ -0,0 +1,240 @@
+import numpy as np
+import pandas as pd
+
+from featuretools.primitives import (
+    MaxConsecutiveFalse,
+    MaxConsecutiveNegatives,
+    MaxConsecutivePositives,
+    MaxConsecutiveTrue,
+    MaxConsecutiveZeros,
+)
+
+
+class TestMaxConsecutiveFalse:
+    primitive = MaxConsecutiveFalse
+
+    def test_regular(self):
+        primitive_instance = self.primitive()
+        primitive_func = primitive_instance.get_function()
+        array = pd.Series([False, False, False, True, True, False, True])
+        assert primitive_func(array) == 3
+
+    def test_all_true(self):
+        primitive_instance = self.primitive()
+        primitive_func = primitive_instance.get_function()
+        array = pd.Series([True, True, True, True])
+        assert primitive_func(array) == 0
+
+    def test_all_false(self):
+        primitive_instance = self.primitive()
+        primitive_func = primitive_instance.get_function()
+        array = pd.Series([False, False, False])
+        assert primitive_func(array) == 3
+
+    def test_with_nan(self):
```

Comment:
Maybe we are already testing with nan even though we would never get this with the `Boolean` logical type.

Reply:
Removed all the nan test cases as they don't make sense.